### PR TITLE
Feature/include xls and csv errors in report

### PR DESCRIPTION
**Pull request by georgiana-b**
Solves #84.

### CODE REVIEW

File: goodtables/compat.py [hunk 1]
```diff
@@ -45,22 +46,28 @@ def iterenc_utf8(data):
 
         iter = chain(first_lines, data)
         iter = iterenc_utf8(iter)
-        reader = csv.reader(iter, dialect=dialect, **kwargs)
-        for row in reader:
-            yield [str(cell, 'utf-8') for cell in row]
+        csv.field_size_limit(20000000)
```
Comment (pwalsh) [inline goodtables/compat.py]: @georgiana-b you cant add a file size limit here. We need to be able to accept huge files, so this is not a solution.

**Comment by pwalsh** [comment]
@akariv can you please review this PR when you can.

File: goodtables/compat.py (hunk 1 above)
Comment (pwalsh) [inline goodtables/compat.py]: right, my mistake, it is a field size limit.

Comment (georgiana-b) [inline goodtables/compat.py]: It's a _field_ size limit that used to have the default value 131072 which wasn't enough for some of the files so I changed it to a number unlikely to be reached.

**Comment by akariv** [comment]
@pwalsh this looks fine to me.

**Comment by pwalsh** [comment]
@akariv thanks

**Pull request merged**